### ISSUE TITLE
window 

### ISSUE THREAD

**Issue by weilenggmj** [sicxu/Deep3DFaceRecon_pytorch#47]
Can I use it by Windows system？

**Comment by Michaelwhite34** [comment]
Yes you can,I just succeeded,the hard part is the installation of nvdiffrast,requiring external compiler for compiling the CUDA kernels.

**Comment by wordsand** [comment]
Download https://github.com/ninja-build/ninja/releases/download/v1.10.2/ninja-win.zip and unzip it to a dir, add that dir to system path.
add  path where the  cl.exe exists , such as  "D:\Visual Studio 2019\VC\Tools\MSVC\14.29.30133\bin\Hostx86\x86"

**Comment by sniperc96** [comment]
> 下载https://github.com/ninja-build/ninja/releases/download/v1.10.2/ninja-win.zip并将其解压缩到一个目录，将该目录添加到系统路径。 添加cl.exe所在路径，如“D:\Visual Studio 2019\VC\Tools\MSVC\14.29.30133\bin\Hostx86\x86”

Hello,The connection you sent has failed,Can you resend one？

**Comment by siyuehua** [comment]
> > 下载[https://github.com/ninja-build/ninja/releases/download/v1.10.2/ninja-win.zip并将其解压缩到一个目录，将该目录添加到系统路径。](https://github.com/ninja-build/ninja/releases/download/v1.10.2/ninja-win.zip%E5%B9%B6%E5%B0%86%E5%85%B6%E8%A7%A3%E5%8E%8B%E7%BC%A9%E5%88%B0%E4%B8%80%E4%B8%AA%E7%9B%AE%E5%BD%95%EF%BC%8C%E5%B0%86%E8%AF%A5%E7%9B%AE%E5%BD%95%E6%B7%BB%E5%8A%A0%E5%88%B0%E7%B3%BB%E7%BB%9F%E8%B7%AF%E5%BE%84%E3%80%82) 添加cl.exe所在路径，如“D:\Visual Studio 2019\VC\Tools\MSVC\14.29.30133\bin\Hostx86\x86”
> 
> Hello,The connection you sent has failed,Can you resend one？

现在链接是可以下载的
